src/main/java/com/minecolonies/coremod/colony/CitizenData.java

```diff
@@ -912,29 +912,29 @@ public InventoryCitizen getInventory()
         return inventory;
     }
 
-    public <R extends IRequestable> IToken createRequest(@NotNull R requested)
+    public <R extends IRequestable> IToken<?> createRequest(@NotNull final R requested)
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove usage of generic wildcard type. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1452)